ubuntudesign/developer.ubuntu.com
# Add target platforms, rename community

## Done

Add target platforms stub and rename community to community resources.

## QA

- Run ./run to get started 
- Make sure you can get to http://127.0.0.1:8015/target-platforms and http://127.0.0.1:8015/community-resources
